onyo/shell_completion/zsh/_onyo

```diff
@@ -133,8 +133,7 @@ _onyo() {
                 args+=(
                     '(- : *)'{-h,--help}'[show this help message and exit]'
                     '(-m --message)'{-m,--message}'[use the given MESSAGE as the commit message]:MESSAGE: '
-                    '(-d --dir -a --asset)'{-d,--dir}'[only remove directories]'
-                    '(-a --asset -d --dir)'{-a,--asset}'[only remove assets]'
+                    '(-r --recursive)'{-r,--recursive}'[recursively remove directories with all their content]'
```

Comment:
Nit: I like making these as terse as possible:
```
recursively remove directories with all their content
```
to
```
remove directories and their contents
```

Reply:
Done